lib/api/controllers/remoteActionsController.js

```diff
@@ -20,7 +20,7 @@ function RemoteActionsController (kuzzle) {
   };
 
   this.init = () => {
-    this.kuzzle.services.list.broker.listen(this.kuzzle.config.queues.remoteActionsQueue, this.onListenCB);
+    this.kuzzle.services.list.broker.listen(kuzzle.config.queues.remoteActionsQueue, this.onListenCB);
```

Comment:
why?
I mean: here, `kuzzle` and `this.kuzzle` point to the same object, but why replacing `this.kuzzle` by `kuzzle` only here and not anywhere else in the controller ?